definitions/npm/chai_v4.x.x/flow_v0.25.0-/chai_v4.x.x.js

```diff
@@ -44,14 +44,15 @@ declare module "chai" {
     within: (start: T & number, finish: T & number, message?: string) => ExpectChain<T>,
 
     instanceof: (constructor: mixed, message?: string) => ExpectChain<T>,
+    instanceOf: (constructor: mixed, message?: string) => ExpectChain<T>,
```

Comment:
Please provide documentation or code asserting your changes, as asked by the PR template. 
I add to go crawl the chai doc to verify.

Reply:
doc for instanceOf is there https://www.chaijs.com/api/bdd/#method_instanceof